src/FriendsManagementClient.ts

```diff
@@ -101,12 +113,12 @@ export class FriendsManagementClient implements FriendsManagementAPI {
         return friends.includes(userId)
     }
 
-    async addAsFriend(userId: SocialId): Promise<void> {
+    async addAsFriend(userId: SocialId, message?: string): Promise<void> {
```

Comment:
I think you'll need to change the type to avoid issues in the client side (kernel) here.
```suggestion
    async addAsFriend(userId: SocialId, message?: string | undefined): Promise<void> {
```